Common/OJS.Common/GlobalConstants.cs

```diff
@@ -132,5 +132,10 @@ public static class Submissions
         {
             public const string SubmissionDownloadFileName = "Submission_{0}.{1}";
         }
+
+        public static class AutocompleteSearchProperties
+        {
+            public const string UserName = "UserName";
```

Comment:
Why is this in GlobalConstants? Also can you use `nameof(UserProfile.UserName)`?

Reply:
I did not want to have hardcoded values and was wondering where to put it. Your suggestion is much better though.